Formula/vectorscan.rb

```diff
@@ -0,0 +1,40 @@
+class Vectorscan < Formula
+  desc "High-performance regular expression matching library"
+  homepage "https://github.com/VectorCamp/vectorscan"
+  url "https://github.com/VectorCamp/vectorscan/archive/refs/tags/vectorscan/5.4.7.tar.gz"
+  sha256 "cd70c2a7bf632b5374083a450019703605520c10c5614a4c12011c99ab8435dd"
+  license "BSD-3-Clause"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ragel" => :build
+  depends_on "pcre"
+
+  conflicts_with "hyperscan", because: "vectorscan installs include files and libraries of the same name"
```

Comment:
```suggestion
```
Let's keep the arch dependency for now. We can remove the arch dependency and add the `conflicts_with` to both formulae in a separate syntax-only PR to avoid wastefully rebuilding formulae.

Reply:
Understood.